runners/basic_runner.py

```diff
@@ -53,11 +53,14 @@ def __str__(self):
 
     def runnable_status(self):
         """
-        Always execute the task if `waf --options=run_always` was used
+        Always execute the test task
+
+        If the run_modified option is present, then only the modified test
+        binaries will be run.
         """
         ret = super(BasicRunner, self).runnable_status()
         if ret == Task.SKIP_ME:
-            if self.generator.bld.has_tool_option('run_always'):
+            if not self.generator.bld.has_tool_option('run_modified'):
```

Comment:
I don't fully understand this change? If the tool option is `false` for `run_modified` the it runs? I can seem to make out what it the intended behavior.